torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -364,6 +368,12 @@ def __init__(
                 # FSDP module directly
                 submodule._fsdp_use_orig_params = use_orig_params
 
+        # Initializes self.process_group, along with rank and world size. This will
```

Comment:
nit: Personally, I do not like explaining what a function/method call does inline like this since this creates redundancy, which can go stale if only one place is updated. The developer should read the docstring for `_init_process_group_state`.

Reply:
[Easy] I recommend changing before landing

Reply:
I mostly want to emphasize the part a couple lines later that mentions this is done before auto wrapping, and the logic for why, which I think is vaulable.

Reply:
Sounds good.